src/components/o-table/_preview.html

```diff
@@ -5,9 +5,9 @@ <h1 class="o-sg-section__section__title" style="color: red;">Attention - WIP</h1
 
   <h2>With Attributes</h2>
 
-  <axa-table
-    caption="Caption"
-    caption-side="bottom"
+  <table is="axa-table"
+    cap="Caption"
```

Comment:
why cap and not caption?

Reply:
It's by intention.
[`HTMLTableElement.caption`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableElement/caption) is an existing DOM property!

Reply:
fair point. Oky doky :)

Reply:
Please see all intrinsic properties of [`HTMLTableElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableElement#Properties).
Thats also the reason I use `headers` and not `thead`, etc.

Reply:
yepp make sense :)